python/ray/tune/stopper.py

```diff
@@ -219,18 +218,6 @@ def stop_all(self):
         return self.has_plateaued() and self._iterations >= self._patience
 
 
-# Deprecate: 1.4
-@Deprecated
```

Comment:
Do we have a similar way to mark an argument to be deprecated?

Reply:
I don't think so - but we could use the `# Deprecate: 1.x` comment for this?